docs/products/opensearch/howto/opensearch-search-and-python.rst

```diff
@@ -0,0 +1,439 @@
+  
+
+We're not adjusting the ``size`` parameter in this tutorial. Therefore, we're using the default value which is 10 results per search.
+
+
+Common queries
+''''''''''''''
+
+In the next section, we cover some of the more common queries. Time to start querying 🔎 
+
+Match query
+-----------
+
+The ``match`` query helps you to find the best matches with multiple search words and is the default option for a full-text search. For example, if you want to find matches that in the ``title`` has "Chilled Tomato".
+This returns results of titles that contain "Chilled" or "Tomato" on it due to DSL defaults to the "or" operator.
+
+.. code-block:: python
+
+       query_body = {
+                      "query": {
+                        "match": {
+                          "title": "Chilled Tomato"
+                        }
+                      }
+                    }
+
+
+You can run yourself the code to explore the ``match`` function. For example, if you want to find out recipes with the name "Spring" on them:
+
+.. code-block:: shell
+
+  python search.py match title Spring
+
+As a result of the Spring search recipes, you'll find:
+
+.. code-block:: shell
+
+  [
+    'Spring Fever ',
+    'Spring Rolls ',
+    'Spring Feeling ',
+    'Spring Fever ',
+    'Spring Rolls ',
+    'Spring Feeling ',
+    'Spring Vegetable Sauté ',
+    'Spring-Onion Cocktail ',
+    'Braised Spring Legumes ',
+    'Asian Spring Rolls '
+  ]
+
+Find out more about `match queries <https://opensearch.org/docs/latest/opensearch/query-dsl/full-text/#match>`_.
+
+Multi match query
+------------------
+One useful query when you want to align the ``match`` query properties but expand it to search in more fields is the ``multi_match`` query. You can add several fields in the ``fields`` property, to search for the ``query`` string across all those fields included in the list.
+
+.. code-block:: python
+
+     query_body = {
+                    "query": {
+                      "multi_match": {
+                        "query": query,
+                        "fields": [field1, field2 ...]
+                      }
+                    }
+                  }
+
+Check out more results for the multi match queries creating your own. You can use our demo with ``multi-match`` keyword followed by the ``fields`` and the ``query``.
+
+::
+
+  python search.py multi-match title ingredients lemon
+
+
+.. seealso::
+
+  Check out more about `multi match query <https://opensearch.org/docs/latest/opensearch/query-dsl/full-text/#multi-match>`_ on the OpenSearch documentation.
+
+Match phrase query
+------------------
+This query can be used to match exact phrases in a field. Where the ``query`` is the phrase that is being searched in a certain field:
+
+.. code-block:: python
+
+     query_body = {
+                    "query": {
+                      "match_phrase": {
+                        field: {
```

Comment:
Agree that "field" should probably be in quotes in this example

Reply:
I am not sure how to this here because field is a variable that is changing, a parameter. It would be more complete if I show it as in the demo but to avoid repetition, it was recommended to remove.

field and query are not strings, they are variables. Having them as strings is confusing imo. Would you have other suggestions?

Reply:
ping @lornajane or @anelook

Reply:
Good explanation, let's leave it as it is